deploy-queue/src/handler/list.rs

```diff
@@ -26,7 +26,7 @@ pub(crate) async fn cells(client: &Pool<Postgres>, environment: Environment) ->
     println!("Known cells for environment {}:", environment);
     for cell in cells {
         println!(
-            "  - {}-{}-{}-{}",
+            "  - {}-{}-{}-cell-{}",
```

Comment:
Sometimes, I think we expose too many internal details to the deploy queue, and it should be something just `env`, and the workflow caller should decide what to put there.
Just a random note, no actions required 😄